Python_DataEngineering/Project_nick-fury.py

```diff
@@ -0,0 +1,47 @@
+# author :- Harsh Vardhan 
+from datetime import datetime
+import pandas as pd
+import json
+import requests
+import mysql.connector as mysql
+import schedule
+import time
+
+def activity ():
+        url = "https://api.tfl.gov.uk/bikepoint" 
+        response = requests.get(url)
+        data = pd.DataFrame(json.loads(response.text))
+        data['time'] = datetime.now()
+        data.drop(['url','$type','placeType','children','childrenUrls'],axis=1,inplace=True)
+        data2=[]
+        data3=[]
+        data4=[]
```

Comment:
rather than naming data1, data2, etc. its better to name things in a more readable way - like nb_bikes_arr, nb_docks_arr etc